src/runtime/storageNG/handle.ts

```diff
@@ -256,3 +256,19 @@ export class SingletonHandle<T extends Referenceable> extends Handle<CRDTSinglet
         e => this.reportUserExceptionInHost(e, this.particle, 'onHandleSync'));
   }
 }
+
+export function handleFor<T extends CRDTTypeRecord>(key: string,
```

Comment:
It's a bit confusing having two handleFor methods. I'd suggest just calling this handleForNG or something like that, so that it's easier to disambiguate between the two of them. We can rename it later.

Reply:
Done